sticker/src/encoder/deprel/relative_position.rs

```diff
@@ -57,20 +57,17 @@ impl SentenceEncoder for RelativePositionEncoder {
     fn encode(&self, sentence: &Sentence) -> Result<Vec<Self::Encoding>, Error> {
         let mut encoded = Vec::with_capacity(sentence.len());
         for idx in 0..sentence.len() {
```

Comment:
Is it possible that the root has another index than `0`?

Reply:
Nope.

Reply:
Then the loop could start at index 1 and the check for root could be omitted?

```rust
for idx in 1..sentence.len() {
    ...
}
```

Reply:
Fixed.